be/src/storage/tablet.h

```diff
@@ -269,6 +269,12 @@ class Tablet : public BaseTablet {
 
     void reset_compaction(CompactionType type);
 
+    bool enable_persistent_index() { return _tablet_meta->enable_persistent_index(); }
```

Comment:
reconsider this naming, this name suggests this method is to enable persistent index, but actually, it gets the value of this variable, change to get_enable_persistent_index, actually enable_persistent_index is a bit improper, cause it's a verb.